bower.json

```diff
@@ -6,7 +6,9 @@
     "Yoast.com"
   ],
   "dependencies": {
-    "yoast-seo": "https://github.com/Yoast/YoastSEO.js.git"
+    "yoast-seo": "https://github.com/Yoast/YoastSEO.js.git",
+    "qtip2": "2.2.1",
+    "tablesorter": "*"
```

Comment:
I think we should definitely freeze the version here.

As far as I can see, this tablesorter is also not maintained very actively. It seems the community has moved to this fork: https://github.com/Mottie/tablesorter. It has much more activity, followers and functionality. We should consider switching to that one as well.